src/Microsoft.VisualStudio.Threading.Analyzers.Tests/MultiAnalyzerTests.cs

```diff
@@ -75,11 +87,12 @@ internal class Child : Parent {
 }
 ";
 
-            this.VerifyCSharpDiagnostic(new[] { test }, hasEntrypoint: false, allowErrors: true);
+            var expected = Verify.CompilerError("CS0535").WithLocation(6, 23).WithMessage("'Parent' does not implement interface member 'A.Foo()'");
```

Comment:
>'Parent' does not implement interface member 'A.Foo()' [](start = 91, length = 54)

Is this going to fail if run on machines configured with non en-US languages?

Reply:
Possibly, I haven't checked. If it's a problem we can look into a few different ways to resolve it.

Reply:
Great. So long as we have an easy way to disable it, that's fine. 
On my PInvoke project we had enough contributors from around the world that it was a real problem because tests wouldn't pass for them. But we can wait for that to happen here before worrying about it then.